### PR TITLE
Prepare release 1.0.0

### PR DESCRIPTION
**Pull request by jsoriano**
Prepare release 1.0.0. Apart of the version changes, in this PR I also include some additional changes that I would like to see in 1.0:
* Added missing tests for the "static" endpoints, used by Kibana to download specific files (mostly readmes and images).
* Update Go to 1.16.7.

### CODE REVIEW

**Comment by exekias** [comment]
Change LGTM, any thoughts on the README file "Running" section? I think we wanted to clearly document the expectations there

**Comment by jsoriano** [comment]
> Change LGTM, any thoughts on the README file "Running" section? I think we wanted to clearly document the expectations there

I checked, and the section about hosting your own package registry is in the package-storage README, and it already mentions that is not supported: https://github.com/elastic/package-storage/#host-own-elastic-package-registry

**Comment by jsoriano** [comment]
Wel, I will add some "for development" in the package-registry readme :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ variations and will also get more complex over time.
 
 ## Running
 
-There are several options to run this.
+There are several options to run this for development purpouses.
```
Comment (mtojek) [inline README.md:109]: nit: typo

**Review by mtojek — APPROVED** [review]
Ah, we can wait for an avalanche of PRs to Package Storage :)

**Comment by ruflin** [comment]
🥳

**Pull request merged**